src/EventStore.Client.Streams/EventStore.Client.Streams.csproj

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Package descriptions should be different and match packages. They should also have an overview of which bits of Event Store they work with and the operations that can be performed